test/utilities.py

```diff
@@ -35,55 +31,27 @@ def assert_pdf_equal(
         test (unittest.TestCase)
         pdf_or_tmpl: instance of `FPDF` or `Template`. The `output` or `render` method will be called on it.
         rel_expected_pdf_filepath (str): relative file path to a PDF file matching the expected output
-        delete (bool): clean up temporary PDF files after performing test
-        generate (bool): only generate `pdf` output to `rel_expected_pdf_filepath` and return. Useful to create new tests.
```

Comment:
Why removing this parameter? I used `generate=True` a lot to build new tests cases in the past weeks

Reply:
You're right, that `generate` should come back.